tests/integration/test_kinesis.py

```diff
@@ -99,19 +86,30 @@ def test_can_put_records_multiple_blob(self):
         )
         # Give it a few seconds for the record to get into the stream.
         time.sleep(10)
-
-        stream = self.client.describe_stream(StreamName=self.stream_name)
-        shard = stream['StreamDescription']['Shards'][0]
-        shard_iterator = self.client.get_shard_iterator(
-            StreamName=self.stream_name, ShardId=shard['ShardId'],
-            ShardIteratorType='TRIM_HORIZON')
-
-        records = self.client.get_records(
-            ShardIterator=shard_iterator['ShardIterator'])
-        self.assertTrue(len(records['Records']) == 2)
-        # Verify that both made it through.
+        records = self.wait_for_stream_data()
+        self.assert_record_data_contains(records, b'foobar', b'barfoo')
+
+    def wait_for_stream_data(self, num_attempts=6, poll_time=10):
+        # Poll until we get records returned from get_records().
```

Comment:
Is there any specific documentation on why we are making the max wait time 1 min? Or is this being chosen because it was working well for that value and waiting a minute is not too bad?

Reply:
> Is there any specific documentation on why we are making the max wait time 1 min? Or is this being chosen because it was working well for that value and waiting a minute is not too bad?

No, completely arbitrary value that I picked that was larger than the previous time of 10 seconds.